src/sdk/models/jobs.ts

```diff
@@ -70,6 +82,7 @@ export interface Job extends IEntityIdentifier<string>, IAuditedEntity {
 	priority?: number;
 	status?: string;
 	execute_on?: number;
+	resets?: JobResetType[];
```

Comment:
Should this exist at the job level, or could it be accessed via `previous_execution`?

Reply:
this is for the next execution